pkg/errors/errors_test.go

```diff
@@ -145,23 +145,23 @@ func Test_NewWithCause(t *testing.T) {
 			args: args{
 				reason: internalServerCause,
 			},
```

Comment:
Should we add tests for the builder?

Reply:
I was going to add the tests, but I stopped: the builder doesn't contain any logic. It just assigns received values to the embedded instance, and for the BUILD it relies on the `NewWithCause` function.

I believe adding tests would be overkill for this object.